apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -872,4 +918,117 @@ public void createBundleWithoutPaging() throws IOException {
     assertEquals(1, bundle.getTotal());
     assertEquals(1, Integer.parseInt(BfdMDC.get("resources_returned_count")));
   }
+
+  /**
+   * Verifies that {@link TransformerUtilsV2#findOrCreateContainedOrganization} sets organization
+   * and eob correctly.
+   */
+  @Test
+  public void findOrCreateContainedOrganizationSetsOrganizationAndEobCorrectly()
```

Comment:
Is this testing the finding or creating part of the method? (Ideally we'd test both cases)

Reply:
Added a unit test for find and a unit test for create